docs/e2e-testing.md

```diff
@@ -0,0 +1,77 @@
+# Marketplace End to End (e2e) Testing
+
+The marketplace operator leverages the [Operator-SDK](https://github.com/operator-framework/operator-sdk/) to create and run it's e2e tests. While the [Operator-SDK's e2e Test Framework](https://github.com/operator-framework/operator-sdk/blob/master/doc/test-framework/writing-e2e-tests.md) provides a number of tools that makes writing e2e tests easier, it is up to the authors of the operator to decide how to implement the tests. This document will cover the approach that the marketplace team has taken and will provide users with enough information to extend the e2e test portfolio.
```

Comment:
`to create and run it's e2e tests`

should be

`to create and run its e2e tests`